stdlib/public/core/Builtin.swift

```diff
@@ -18,49 +18,26 @@ import SwiftShims
 /// Returns the contiguous memory footprint of `T`.
 ///
 /// Does not include any dynamically-allocated or "remote" storage.
-/// In particular, `sizeof(X.self)`, when `X` is a class type, is the
+/// In particular, `MemoryLayout<>.size`, when `X` is a class type, is the
 /// same regardless of how many stored properties `X` has.
 @_transparent
-public func sizeof<T>(_:T.Type) -> Int {
+func _sizeof<T>(_:T.Type) -> Int {
```

Comment:
My suggestion is to keep these functions as public deprecated, so that we can land this PR without updating corelibs.